src/client/src/pages/overview/sidePanelContent/importer/importModalContent.tsx

```diff
@@ -23,12 +23,22 @@ const ExampleHeadings = (headings: string) => {
   );
 };
 
-const ImportModalContent = ({ setSelectedFile }: ImportContentProps) => {
+const ImportModalContent = ({
+  setSelectedFile,
+  setFileType,
+  fileType,
+}: ImportContentProps & { setFileType: (type: string) => void } & { fileType: string | null }) => {
```

Comment:
REQ: Eigenes Interface für diese Komponente erstellen, am besten gleich in diesem File. (ImportContentProps braucht es glaube ich nicht mehr, da nur 1 Interface davon erbt).